### PR TITLE
Display multiple images on the same board

### PR DESCRIPTION
**Pull request by chaodaiG**
yaml driven insertions

### CODE REVIEW

File: hackathon/pkg/imagecompiler/imagecompiler.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+			if insertionDef.MaxHeight == 0 {
+				height = instImg.Bounds().Dy() * insertionDef.MaxWidth / instImg.Bounds().Dx()
+			}
+			resized := resize.Resize(uint(width), uint(height), instImg, resize.Lanczos3)
+			instImg = renderImage(resized, dither)
+		}
+		coord := insertionDef.toPoint()
+		bounds := instImg.Bounds()
+		r := bounds.Sub(bounds.Min).Add(*coord)
+		draw.Draw(bgHackImage, r, instImg, *coord, draw.Src)
```
Comment (chaodaiG) [inline hackathon/pkg/imagecompiler/imagecompiler.go]: Something is wrong here, but I couldn't figure out why. The desired behavor is:
```
10 X 10.              20 X 20.            40 X 40
                       20 X 20.           40 X 40
                                          40 X 40
                                          40 X 40

80                            X                          80
80                            X                          80
80                            X                          80
80                            X                          80
80                            X                          80
80                            X                          80
80                            X                          80
80                            X                          80
```

but got
Screen Shot 2021-06-16 at 1 05 50 PM

Which seems to me the `draw.Draw` function redraws from `Point{0, 0}` every time, while still understands that the drawing starts from `coord`, so truncated the portion on the left for first row

@cjwagner @fejta , do you guys know why?

Comment (chaodaiG) [reply hackathon/pkg/imagecompiler/imagecompiler.go]: The values when processing `20X20`:
```
coords: (12,0)
bounds:  (0,0)-(20,20)
r:  (12,0)-(32,20)
instImg bounds:  (0,0)-(20,20)
```

Comment (cjwagner) [reply hackathon/pkg/imagecompiler/imagecompiler.go]: Try replacing `*coord` with `bounds.Min` on line 149. I agree that this behavior is confusing and don't really understand why it is the way it, but it is.

Comment (chaodaiG) [reply hackathon/pkg/imagecompiler/imagecompiler.go]: that's exactly it!

**Pull request merged**